pkg/osquery/tables/launcher_db/launcher_db.go

```diff
@@ -1,31 +1,28 @@
-package kolide_server_data
```

Comment:
I do not think this is a good idea. 

Two reasons.

First, this table should be here, standing alone. The intent is not to debug, but to provide a hook for queries to join against.

Second, I think a launcher_db table would need an allowlist of buckets to iterate. There are buckets and keys that should not be exposed